.github/workflows/release.yaml

```diff
@@ -7,28 +7,28 @@ on:
         description: Commit SHA, tag or branch name
         required: true
       kuadrantOperatorVersion:
-        description: Kuadrant Operator version
-        default: 0.0.0
+        description: Kuadrant Operator version (excluding the v)
+        default: "200.20.2-replace"
```

Comment:
What is "200.20.2-replace"? I'm guess it's meant to work a better example of a semantic version than "0.0.0" perhaps.

Reply:
yes exactly